VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/services/vho-query-service.service.ts

```diff
@@ -49,12 +57,33 @@ export class VhoQueryService {
         this.interval = setInterval(async () => {
             await this.runQuery();
         }, this.pollingInterval);
+        this.eventService
+            .getHearingDetailsUpdated()
+            .pipe(takeUntil(this.destroy$))
+            .subscribe(hearingDetailMessage => this.handleHearingDetailUpdate(hearingDetailMessage));
+    }
+
+    handleHearingDetailUpdate(hearingDetailMessage: HearingDetailsUpdatedMessage) {
+        if (hearingDetailMessage.conference) {
+            const newConference = hearingDetailMessage.conference;
+            const index = this.vhoConferences.findIndex(x => x.id === newConference.id);
+            const originalConference = this.vhoConferences[index];
+            if (index !== -1) {
+                const newHearing = this.mapConferenceResponseToConferenceForVhOfficerResponse(newConference);
```

Comment:
I don't believe you need a mapper here. you can achieve the same result as:
```
let foundConference = this.vhoConferences[foundIndex];
foundConference = new ConferenceForVhOfficerResponse({
                    ...foundConference,
                    case_name: conference.case_name,
                    case_number: conference.case_number,
                    participants: conference.participants
                });
this.vhoConferences[foundIndex] = foundConference;
```

1. You don't have mislead representation of the same object type by calling one conference and the other a hearing
2. You retain all the original properties and override only what you care to (i.e. no need to re-assign `allocated_cso`

Reply:
done